src/client/components/Swipe/Swipe.jsx

```diff
@@ -312,20 +317,33 @@ class Swipe extends Component {
     );
   }
 
-  renderSwipeComplete(listBuffer) {
+  renderSwipeComplete() {
     if (this.state.initialSetup === false) {
-      if (listBuffer && (listBuffer.attractions.length > 0 || listBuffer.food.length > 0)) {
+      if (this.state.listBuffer && (this.state.listBuffer.attractions.length > 0 || this.state.listBuffer.food.length > 0)) {
         this.setState({ initialSetup: true });
       }
     }
     return (
       <div className="swipe-container">
         <div className="center-align">
-          <div className="no-card-msg">We're looking for more places</div>
+          {
+            ( (this.state.swipeList === 2 && this.state.attractionsHasNext) ||
+              (this.state.swipeList === 1 && this.state.foodHasNext) ) &&
+              <div className="no-card-msg">We're looking for more places!</div>
+          }
+          {
+            ( (this.state.swipeList === 2 && !this.state.attractionsHasNext) ||
+              (this.state.swipeList === 1 && !this.state.foodHasNext) ) &&
+              <div className="no-card-msg">No more locations in this list!</div>
+          }
```

Comment:
I think these conditions can be written more readably like this?
```suggestion
          <div className="no-card-msg">
            { this.state.swipeList === 2
              ? this.state.attractionsHasNext
                ? "We're looking for more places!"
                : "No more locations in this list!"
              : this.state.foodHasNext
                ? "We're looking for more places!"
                : "No more locations in this list!"
              }
            </div>
```